server/auth_server.go

```diff
@@ -503,11 +504,25 @@ func UpdateConfig(authConfig model.AuthConfig) error {
 	commonSettings[providerNameSetting] = authConfig.Provider
 	commonSettings[providerSetting] = authConfig.Provider
 	commonSettings[externalProviderSetting] = "true"
+	err = updateCommonSettings(commonSettings)
```

Comment:
@cjellick the call to update the setting `api.security.enabled` should be after all other settings are updated without errors. That's how it was in the earlier code. But the call to update timestamp needn't be separate. I'll change this